### ISSUE TITLE
Sourcemaps relative to gulp.src base option

### ISSUE THREAD

**Issue by at0g** [stevelacy/gulp-stylus#111]
Hello,

I'm experiencing some unexpected behaviour with sourcemaps from gulp-stylus.

The issue occurs when there is a `base` option on `gulp.src` and external sourcemaps are created.
(The sourcemaps refer to *.css instead of *.styl)

```
gulp.src('./in/css/*.styl', { base: './in' } )
.pipe(...)
.pipe(gulp.dest('./in')
```

But does not happen with this config:
```
gulp.src('./in/css/*.styl', { base: './in/css' } )
.pipe(...)
.pipe(gulp.dest('./out/css')
``` 

Please take a look at this repo to reproduce the issue:
https://github.com/at0g/gulp-stylus-sourcemaps-issue-example

The above repo also shows other permutations (such as css only, no stylus) where the issue does not occur.

**Comment by stevelacy** [comment]
Please update gulp-stylus to v2.0.0 and post the results

**Comment by at0g** [comment]
The example is using gulp-stylus 2.0:
https://github.com/at0g/gulp-stylus-sourcemaps-issue-example/blob/master/package.json#L15

**Comment by stevelacy** [comment]
Published to 2.0.1, thanks

**Comment by at0g** [comment]
Thanks for the plugin :+1:
